packages/itwinui-css/src/style/global.scss

```diff
@@ -65,3 +65,10 @@ html {
   // Scrollbar
   @include iui-scrollbar;
 }
+
+[class*='iui-'],
+[class*='iui-'] *,
```

Comment:
hm... when thinking about it, I dont feel like we can just apply border-box to all children of iui elements.. That ends up in all children of `iui-body` having border-box and thats guaranteed to mess up users projects..
Maybe lets tackle ONLY iui elements? See which elements are getting broken when changing this..
Also, I think we might want to exclude body from iui selector, because body is also a tricky part to change box-sizing :/
This change also should fix some of the test images.

Reply:
Opinions @FlyersPh9 @mayank99 @bentleyvk  ?

Reply:
Yeah, we can't target all children either target only `iui-*` or we should omit body by adding `:not(.iui-body)` but it will increase specificity.

Reply:
I updated this and removed `*` for now.

Reply:
Would this work?
```scss
[class*='iui-'],
[class*='iui-']::before,
[class*='iui-']::after { … }
```